app/src/main/java/com/paypal/psix/activities/SettingsActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.paypal.psix.activities;
+
+import android.os.Bundle;
+import android.preference.Preference;
+import android.preference.PreferenceActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import com.paypal.psix.R;
+
+public class SettingsActivity extends PreferenceActivity
+        implements Preference.OnPreferenceChangeListener {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        addPreferencesFromResource(R.xml.pref_general);
+
+        setupToolbar();
+        setupSignOut();
+    }
+
+    private void setupToolbar() {
+        LinearLayout root = (LinearLayout)findViewById(android.R.id.list).getParent().getParent().getParent();
+        Toolbar bar = (Toolbar)LayoutInflater.from(this).inflate(R.layout.settings_bar, root, false);
+        root.addView(bar, 0); // insert at top
+        bar.setNavigationOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                finish();
+            }
+        });
+    }
+
+    @Override
+    public boolean onPreferenceChange(Preference preference, Object newValue) {
+        return true;
+    }
+
+    private void setupSignOut() {
+        Preference button = findPreference(getString(R.string.pref_signout_key));
```

Comment:
nope, doesn't support preferences.
there's an issue for this on their repo and they say they want to keep it focused and not "INJECT EVERYTHING!!!!11"